src/components/PlsIDForm.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { Form } from "library-simplified-reusable-components";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+import { FetchErrorData } from "opds-web-client/lib/interfaces";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibraryData } from "../interfaces";
+import { CheckSoloIcon } from "@nypl/dgx-svg-icons";
+import Input from "./reusables/Input";
+
+export interface PlsIDFormOwnProps {
+  uuid: string;
+  store: Store<State>;
+  fetchLibrary: (uuid: string) => LibraryData;
+  current?: string;
+}
+
+export interface PlsIDFormStateProps {
+  error?: FetchErrorData;
+}
+
+export interface PlsIDFormDispatchProps {
+  plsID?: (data: FormData) => Promise<void>;
+}
+
+export interface PlsIDFormProps extends PlsIDFormOwnProps, PlsIDFormStateProps, PlsIDFormDispatchProps {}
+
+export interface PlsIDState {
+  saved: boolean;
+}
+
+export class PlsIDForm extends React.Component<PlsIDFormProps, PlsIDState> {
+  constructor(props: PlsIDFormProps) {
+    super(props);
+    this.state = { saved: false };
+    this.submit = this.submit.bind(this);
+  }
+  async submit(data: FormData) {
+    await this.props.plsID(data);
```

Comment:
Maybe a variable name like `postPlsID` would be helpful. I originally thought it was fetching a pls id.

Reply:
Also to go along with other verb-y action names.